model.py

```diff
@@ -3722,6 +3722,7 @@ def set_presentation_ready_based_on_content(self):
         """
         if (not self.primary_edition
             or not self.license_pools
+            or all([lp.suppressed for lp in self.license_pools])
```

Comment:
You're right, I'll take this out. For some reason I was thinking I needed it but works with suppressed license pools are already left out by other code.